generator/internal/dart/templates/lib/message.mustache

```diff
@@ -19,7 +19,15 @@ limitations under the License.
 /// {{{.}}}
 {{/Codec.DocLines}}
 class {{Codec.Name}} {
-  // TODO:
+  {{#Fields}}
+  {{> field}}
+  {{/Fields}}
+
+  {{Codec.Name}}({
```

Comment:
Is this a constructor-like thing?  Would customers expect a comment?

Reply:
We should doc if we can provide something more than:

```
/// Creates a [Foo].
class Foo {
  ...
```

Perhaps that would be:

```
/// Creates a [Foo].
///
/// [paramOne]: Lorem ipsum dolor sit amet...
///
/// [paramTwo]: Lorem ipsum dolor sit amet...
class Foo {
  ...
```

That would probably work pretty well, esp. if we used the first markdown sentence from the field docs to construct the ctor doc.